src/components/chat/SuggestionMembers.tsx

```diff
@@ -0,0 +1,78 @@
+import { FC } from 'react';
+import { Pressable, StyleSheet, Text, View } from 'react-native';
+
+import { Member, UUID } from '@graasp/sdk';
+
+import { PRIMARY_LIGHT_COLOR } from '../../config/constants/constants';
+import { useQueryClient } from '../../context/QueryClientContext';
+
+interface SuggestionMembersProps {
+  itemId: UUID;
+  keyword: any;
+  onSuggestionPress: any;
+}
+
+const SuggestionMembers: FC<SuggestionMembersProps> = ({
+  itemId,
+  keyword,
+  onSuggestionPress,
+}) => {
+  const { hooks } = useQueryClient();
+  const {
+    data: itemMemberships,
+    isLoading: isLoadingItemMemberships,
+    isError: isErrorItemMemberships,
+  } = hooks.useItemMemberships(itemId);
+
+  if (isLoadingItemMemberships || !itemMemberships) {
+    return null;
+  }
+  if (isErrorItemMemberships) {
+    console.error('Error in SuggestionMembers');
+    return null;
+  }
```

Comment:
Have these checks after the rendering function (that will return on `if(itemMemberships)`)